src/main/java/lottogame/domain/LottoGame.java

```diff
@@ -0,0 +1,36 @@
+package lottogame.domain;
+
+import java.util.EnumMap;
+import java.util.Map;
+import lottogame.domain.number.LottoWinningNumbers;
+import lottogame.domain.ticket.LottoTickets;
+
+public class LottoGame {
+
+    private static final int INIT_COUNT = 0;
+
+    private final LottoTickets lottoTickets;
+    private final LottoWinningNumbers lottoWinningNumbers;
+
+    public LottoGame(final LottoTickets lottoTickets,
+        final LottoWinningNumbers lottoWinningNumbers) {
```

Comment:
```suggestion
    public LottoGame(final LottoTickets lottoTickets, final LottoWinningNumbers lottoWinningNumbers) {
```
줄바꿈을 하신 특별한 이유가 있을까요? 🤔

Reply:
가로로 너무 길어지는 것 같아서 개행을 했었어요!